tests/test_layer.py

```diff
@@ -1,12 +1,10 @@
 """Test layer.py"""
-import pytest
-import torch
-import math
-
-import numpy as np
-
 from hypothesis import given
 import hypothesis.strategies as st
+import math
```

Comment:
`math` is a system import so it should go at its own section at the top. Next come third-party imports (`pytest`, `torch`, `bumpy`, `hypothesis`, and then finally the leabra7 stuff. See https://github.com/Khan/style-guides/blob/master/style/python.md#import-style